src/Pages/ChannelRequests.razor

```diff
@@ -165,7 +165,7 @@
             </DataGridColumn>
             <DataGridCommandColumn TItem="ChannelOperationRequest" Caption="Actions" Displayable="true">
                 <EditCommandTemplate>
-                    <Button Color="Color.Primary" hidden=@_isNodeManager Clicked="@(() => ShowModal(context.Item))">Approve</Button>
+                    <Button Color="Color.Primary" hidden=@(!_isFinanceManager) Clicked="@(() => ShowModal(context.Item))">Approve</Button>
```

Comment:
? Why is this in this PR 🤔

Reply:
That's the small bug I caught that I mentioned. I can separate it to other PR if needed

Reply:
Its ok